class.phpmailer.php

```diff
@@ -1364,7 +1364,8 @@ public function postSend()
      */
     protected function sendmailSend($header, $body)
     {
-        if (!empty($this->Sender)) {
+        // CVE-2016-10033, CVE-2016-10045: Don't pass -f if characters will be escaped by escapeshellcmd when popen is called due to safe mode.
+        if (!empty($this->Sender) || ini_get('safe_mode')) {
```

Comment:
Yes, it should be.  I have a pull request on the way; I'm just checking something else while I'm at it.